README.md

```diff
@@ -66,6 +66,58 @@ you can use `[headers]` directive like this.
 
 `[clearButtonCaption]="'Clear'"` - Text shown on the "Clear" button. Default is "**Clear**".
 
+#### CSS Class
+
+`[class]="'customClass'"` - Set custom class for this component. It is set on parent level element `<image-upload>` which can be customized as follow :
+
+    .custom-class{    
+        background-color: #dd3;
+        border-radius: 5px;
+        margin:5px;
+        width: 500px;        
+    }
+    .custom-class .img-ul-upload{
+        background-color: #000 !important;
+    }
+    .custom-class .img-ul-clear{
+        background-color: #B819BB !important;
+    }
+    .custom-class .img-ul-drag-box-msg {
+        color: purple !important;
+    }
+    .custom-class .img-ul-container{
+        background-color: #FF6CAD !important;
+    }
+    
+**Note:** 
+- `.img-ul-*` is class overridden with new styles. 
+- Also set `encapsulation: ViewEncapsulation.Native/None` in your component.
```

Comment:
It was for avoiding scope to override.  Needn't be required more because  "deep" selector (>>>) performs well.